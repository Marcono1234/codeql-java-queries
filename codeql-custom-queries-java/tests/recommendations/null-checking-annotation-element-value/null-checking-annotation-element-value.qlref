./recommendations/null-checking-annotation-element-value.ql
