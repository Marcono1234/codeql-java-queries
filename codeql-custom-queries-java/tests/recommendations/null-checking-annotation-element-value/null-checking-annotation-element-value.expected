| Test.java:8:16:8:32 | ... == ... | Redundant null check for annotation element value obtained from $@ | Test.java:8:16:8:24 | value(...) | here |
| Test.java:8:37:8:53 | ... != ... | Redundant null check for annotation element value obtained from $@ | Test.java:8:37:8:45 | value(...) | here |
| Test.java:13:16:13:24 | ... == ... | Redundant null check for annotation element value obtained from $@ | Test.java:12:20:12:28 | value(...) | here |
| Test.java:13:29:13:37 | ... != ... | Redundant null check for annotation element value obtained from $@ | Test.java:12:20:12:28 | value(...) | here |
| Test.java:18:16:18:32 | ... == ... | Redundant null check for annotation element value obtained from $@ | Test.java:18:16:18:24 | array(...) | here |
| Test.java:22:16:22:35 | ... == ... | Redundant null check for annotation element value obtained from $@ | Test.java:22:16:22:24 | array(...) | here |
| Test.java:22:40:22:59 | ... != ... | Redundant null check for annotation element value obtained from $@ | Test.java:22:40:22:48 | array(...) | here |
| Test.java:28:16:28:24 | ... == ... | Redundant null check for annotation element value obtained from $@ | Test.java:26:26:26:34 | array(...) | here |
| Test.java:28:29:28:37 | ... != ... | Redundant null check for annotation element value obtained from $@ | Test.java:26:26:26:34 | array(...) | here |
| Test.java:35:16:35:24 | ... == ... | Redundant null check for annotation element value obtained from $@ | Test.java:34:13:34:21 | value(...) | here |
